finalPrep/README.md

```diff
@@ -18,37 +22,45 @@ Some things to consider as you refactor include...
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+<<<<<<< HEAD
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
 - `Add Week 2 Screenshot Here`
 - `Add Week 3 Screenshot Here`
+=======
+
+- <img width="1029" alt="Screen Shot 2021-04-24 at 10 54 52 AM" src="https://user-images.githubusercontent.com/79541611/115970452-72433900-a510-11eb-862e-02acd1255cfc.png">
+
+- <img width="1049" alt="Screen Shot 2021-04-24 at 10 55 36 AM" src="https://user-images.githubusercontent.com/79541611/115970464-7d966480-a510-11eb-8230-a1cd106077a8.png">
+
+- <img width="1035" alt="Screen Shot 2021-04-24 at 10 55 50 AM" src="https://user-images.githubusercontent.com/79541611/115970470-838c4580-a510-11eb-8c3f-bfde8b683d7c.png">
+
+>>>>>>> 74306bf55b9ef0f40889d5cae710f2239fd104ef
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
-
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: <script src="https://gist.github.com/Samantha-Brown/47b7eaab0cb45228261a419710e0bbf6.js"></script>
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
-
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add link to your gist here:<script src="https://gist.github.com/Samantha-Brown/e8ea82f0391831a6178b2c188f1d9e01.js"></script>
```

Comment:
✅ 
🚨 Make sure you know how to make Markdown hyperlinks inside your text. If you check out this readme [live](https://github.com/Samantha-Brown/frontend-mod-1-prework/tree/main/finalPrep#mentorship-prep) you will see that your links aren't quite working. HINT - do you need the script syntax?